### ISSUE TITLE
Setting angle from limb too large causes a crash

### ISSUE THREAD

**Issue by aludez** [NuSpaceSim/nuSpaceSim#17]
It seems to depend on detector altitude setting as well.
When DetectorAltitude = 30 km, the crash happens when AngleFromLimb is set larger than 30 degrees.
With DetectorAltitude = 525 km, it happens when AngleFromLimb is set larger than 25 degrees.
The error I get is pasted below:
```
Traceback (most recent call last):
  File "/home/abl/.local/bin/nuspacesim", line 33, in <module>
    sys.exit(load_entry_point('nuspacesim', 'console_scripts', 'nuspacesim')())
  File "/home/abl/.local/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/home/abl/.local/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/home/abl/.local/lib/python3.8/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/abl/.local/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/abl/.local/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/abl/nuspacesim/src/nuspacesim/apps/cli.py", line 184, in run
    simulation = compute(
  File "/home/abl/nuspacesim/src/nuspacesim/compute.py", line 175, in compute
    tauBeta, tauLorentz, tauEnergy, showerEnergy, tauExitProb = tau(
  File "/home/abl/nuspacesim/src/nuspacesim/utils/decorators.py", line 199, in wrapper_f
    values = func(*args, **kwargs)
  File "/home/abl/nuspacesim/src/nuspacesim/utils/decorators.py", line 82, in store_f
    values = func(*args, **kwargs)
  File "/home/abl/nuspacesim/src/nuspacesim/simulation/taus/taus.py", line 172, in __call__
    tauExitProb = self.tau_exit_prob(betas, log_e_nu)
  File "/home/abl/nuspacesim/src/nuspacesim/simulation/taus/taus.py", line 115, in tau_exit_prob
    Pexit[mask] = 10 ** pexit_interp((log_e_nu[mask], betas[mask]))
  File "/home/abl/.local/lib/python3.8/site-packages/scipy/interpolate/interpolate.py", line 2517, in __call__
    raise ValueError("One of the requested xi is out of bounds "
ValueError: One of the requested xi is out of bounds in dimension 1
```

**Comment by Areustle** [comment]
I see this too. It seems to be caused by `beta_tr`'s above the upper bound for the table (42 degrees)
```
In [8]: np.amax(np.degrees(r['beta_rad']))
Out[8]: 43.347248866910284

In [9]: np.degrees(r['beta_rad'])[np.degrees(r['beta_rad']) >= 42]
Out[9]: 
<Column name='beta_rad' dtype='float64' length=299>
 42.77202613819143
 42.26195568621552
...```

**Comment by Areustle** [comment]
I suspect the correct behavior here is to return 0 for those pexit and tau_energy values.

**Comment by Areustle** [comment]
So of the three options to address this we have

1. Extend the table value from 42 onwards (the same thing we do with betas below 1 degree).
2. Set a low-but-not-zero default value.
3. Invalidate high-beta events.

And we’re choosing #3.
@sameer-patel-1 suggests the statistical fidelity of the current tables at those high beta angles is just too low to rely on their extension, and as we saw earlier we’re concerned about the downstream consequences of option #2.

This issue is addressed in commit a35386428f1ef9dda8c38407b968a4f2b8d6a9db
